Traceback (most recent call last):
  File "c:\users\samuel\miniconda3\envs\tensorflow\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "c:\users\samuel\miniconda3\envs\tensorflow\lib\site-packages\nbclient\client.py", line 1082, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\users\samuel\miniconda3\envs\tensorflow\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\users\samuel\miniconda3\envs\tensorflow\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "c:\users\samuel\miniconda3\envs\tensorflow\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "c:\users\samuel\miniconda3\envs\tensorflow\lib\site-packages\nbclient\client.py", line 536, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "c:\users\samuel\miniconda3\envs\tensorflow\lib\site-packages\nbclient\client.py", line 827, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "c:\users\samuel\miniconda3\envs\tensorflow\lib\site-packages\nbclient\client.py", line 735, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from numpy import array
from keras.models import Sequential
from keras.layers import LSTM
from keras.layers import Dense
 
# split a univariate sequence into samples
def split_sequence(sequence, n_steps):
	X, y = list(), list()
	for i in range(len(sequence)):
		# find the end of this pattern
		end_ix = i + n_steps
		# check if we are beyond the sequence
		if end_ix > len(sequence)-1:
			break
		# gather input and output parts of the pattern
		seq_x, seq_y = sequence[i:end_ix], sequence[end_ix]
		X.append(seq_x)
		y.append(seq_y)
	return array(X), array(y)
 
# define input sequence
raw_seq = [10, 20, 30, 40, 50, 60, 70, 80, 90]
# choose a number of time steps
n_steps = 6
# split into samples
X, y = split_sequence(raw_seq, n_steps)
# reshape from [samples, timesteps] into [samples, timesteps, features]
n_features = 1
X = X.reshape((X.shape[0], X.shape[1], n_features))
# define model
model = Sequential()
model.add(LSTM(50, activation='relu', input_shape=(n_steps, n_features)))
model.add(Dense(1))
model.compile(optimizer='adam', loss='mse')
# fit model
model.fit(X, y, epochs=200, verbose=0)
# demonstrate prediction
x_input = array([40, 50 ,60, 70, 80, 90])
x_input = x_input.reshape((1, len(x_input), n_features))
yhat = model.predict(x_input, verbose=0)
print(yhat)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mInternalError[0m                             Traceback (most recent call last)
[1;32m<ipython-input-1-61103b65a025>[0m in [0;36m<module>[1;34m[0m
[0;32m     34[0m [0mmodel[0m[1;33m.[0m[0mcompile[0m[1;33m([0m[0moptimizer[0m[1;33m=[0m[1;34m'adam'[0m[1;33m,[0m [0mloss[0m[1;33m=[0m[1;34m'mse'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m     35[0m [1;31m# fit model[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[1;32m---> 36[1;33m [0mmodel[0m[1;33m.[0m[0mfit[0m[1;33m([0m[0mX[0m[1;33m,[0m [0my[0m[1;33m,[0m [0mepochs[0m[1;33m=[0m[1;36m200[0m[1;33m,[0m [0mverbose[0m[1;33m=[0m[1;36m0[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m     37[0m [1;31m# demonstrate prediction[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[0;32m     38[0m [0mx_input[0m [1;33m=[0m [0marray[0m[1;33m([0m[1;33m[[0m[1;36m40[0m[1;33m,[0m [1;36m50[0m [1;33m,[0m[1;36m60[0m[1;33m,[0m [1;36m70[0m[1;33m,[0m [1;36m80[0m[1;33m,[0m [1;36m90[0m[1;33m][0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\miniconda3\envs\tensorflow\lib\site-packages\keras\engine\training.py[0m in [0;36mfit[1;34m(self, x, y, batch_size, epochs, verbose, callbacks, validation_split, validation_data, shuffle, class_weight, sample_weight, initial_epoch, steps_per_epoch, validation_steps, validation_freq, max_queue_size, workers, use_multiprocessing, **kwargs)[0m
[0;32m   1237[0m                                         [0msteps_per_epoch[0m[1;33m=[0m[0msteps_per_epoch[0m[1;33m,[0m[1;33m[0m[1;33m[0m[0m
[0;32m   1238[0m                                         [0mvalidation_steps[0m[1;33m=[0m[0mvalidation_steps[0m[1;33m,[0m[1;33m[0m[1;33m[0m[0m
[1;32m-> 1239[1;33m                                         validation_freq=validation_freq)
[0m[0;32m   1240[0m [1;33m[0m[0m
[0;32m   1241[0m     def evaluate(self,

[1;32m~\miniconda3\envs\tensorflow\lib\site-packages\keras\engine\training_arrays.py[0m in [0;36mfit_loop[1;34m(model, fit_function, fit_inputs, out_labels, batch_size, epochs, verbose, callbacks, val_function, val_inputs, shuffle, initial_epoch, steps_per_epoch, validation_steps, validation_freq)[0m
[0;32m    194[0m                     [0mins_batch[0m[1;33m[[0m[0mi[0m[1;33m][0m [1;33m=[0m [0mins_batch[0m[1;33m[[0m[0mi[0m[1;33m][0m[1;33m.[0m[0mtoarray[0m[1;33m([0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m    195[0m [1;33m[0m[0m
[1;32m--> 196[1;33m                 [0mouts[0m [1;33m=[0m [0mfit_function[0m[1;33m([0m[0mins_batch[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    197[0m                 [0mouts[0m [1;33m=[0m [0mto_list[0m[1;33m([0m[0mouts[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m    198[0m                 [1;32mfor[0m [0ml[0m[1;33m,[0m [0mo[0m [1;32min[0m [0mzip[0m[1;33m([0m[0mout_labels[0m[1;33m,[0m [0mouts[0m[1;33m)[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\miniconda3\envs\tensorflow\lib\site-packages\tensorflow_core\python\keras\backend.py[0m in [0;36m__call__[1;34m(self, inputs)[0m
[0;32m   3725[0m         [0mvalue[0m [1;33m=[0m [0mmath_ops[0m[1;33m.[0m[0mcast[0m[1;33m([0m[0mvalue[0m[1;33m,[0m [0mtensor[0m[1;33m.[0m[0mdtype[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m   3726[0m       [0mconverted_inputs[0m[1;33m.[0m[0mappend[0m[1;33m([0m[0mvalue[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[1;32m-> 3727[1;33m     [0moutputs[0m [1;33m=[0m [0mself[0m[1;33m.[0m[0m_graph_fn[0m[1;33m([0m[1;33m*[0m[0mconverted_inputs[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m   3728[0m [1;33m[0m[0m
[0;32m   3729[0m     [1;31m# EagerTensor.numpy() will often make a copy to ensure memory safety.[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\miniconda3\envs\tensorflow\lib\site-packages\tensorflow_core\python\eager\function.py[0m in [0;36m__call__[1;34m(self, *args, **kwargs)[0m
[0;32m   1549[0m       [0mTypeError[0m[1;33m:[0m [0mFor[0m [0minvalid[0m [0mpositional[0m[1;33m/[0m[0mkeyword[0m [0margument[0m [0mcombinations[0m[1;33m.[0m[1;33m[0m[1;33m[0m[0m
[0;32m   1550[0m     """
[1;32m-> 1551[1;33m     [1;32mreturn[0m [0mself[0m[1;33m.[0m[0m_call_impl[0m[1;33m([0m[0margs[0m[1;33m,[0m [0mkwargs[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m   1552[0m [1;33m[0m[0m
[0;32m   1553[0m   [1;32mdef[0m [0m_call_impl[0m[1;33m([0m[0mself[0m[1;33m,[0m [0margs[0m[1;33m,[0m [0mkwargs[0m[1;33m,[0m [0mcancellation_manager[0m[1;33m=[0m[1;32mNone[0m[1;33m)[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\miniconda3\envs\tensorflow\lib\site-packages\tensorflow_core\python\eager\function.py[0m in [0;36m_call_impl[1;34m(self, args, kwargs, cancellation_manager)[0m
[0;32m   1589[0m       raise TypeError("Keyword arguments {} unknown. Expected {}.".format(
[0;32m   1590[0m           list(kwargs.keys()), list(self._arg_keywords)))
[1;32m-> 1591[1;33m     [1;32mreturn[0m [0mself[0m[1;33m.[0m[0m_call_flat[0m[1;33m([0m[0margs[0m[1;33m,[0m [0mself[0m[1;33m.[0m[0mcaptured_inputs[0m[1;33m,[0m [0mcancellation_manager[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m   1592[0m [1;33m[0m[0m
[0;32m   1593[0m   [1;32mdef[0m [0m_filtered_call[0m[1;33m([0m[0mself[0m[1;33m,[0m [0margs[0m[1;33m,[0m [0mkwargs[0m[1;33m)[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\miniconda3\envs\tensorflow\lib\site-packages\tensorflow_core\python\eager\function.py[0m in [0;36m_call_flat[1;34m(self, args, captured_inputs, cancellation_manager)[0m
[0;32m   1690[0m       [1;31m# No tape is watching; skip to running the function.[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[0;32m   1691[0m       return self._build_call_outputs(self._inference_function.call(
[1;32m-> 1692[1;33m           ctx, args, cancellation_manager=cancellation_manager))
[0m[0;32m   1693[0m     forward_backward = self._select_forward_and_backward_functions(
[0;32m   1694[0m         [0margs[0m[1;33m,[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\miniconda3\envs\tensorflow\lib\site-packages\tensorflow_core\python\eager\function.py[0m in [0;36mcall[1;34m(self, ctx, args, cancellation_manager)[0m
[0;32m    543[0m               [0minputs[0m[1;33m=[0m[0margs[0m[1;33m,[0m[1;33m[0m[1;33m[0m[0m
[0;32m    544[0m               [0mattrs[0m[1;33m=[0m[1;33m([0m[1;34m"executor_type"[0m[1;33m,[0m [0mexecutor_type[0m[1;33m,[0m [1;34m"config_proto"[0m[1;33m,[0m [0mconfig[0m[1;33m)[0m[1;33m,[0m[1;33m[0m[1;33m[0m[0m
[1;32m--> 545[1;33m               ctx=ctx)
[0m[0;32m    546[0m         [1;32melse[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0;32m    547[0m           outputs = execute.execute_with_cancellation(

[1;32m~\miniconda3\envs\tensorflow\lib\site-packages\tensorflow_core\python\eager\execute.py[0m in [0;36mquick_execute[1;34m(op_name, num_outputs, inputs, attrs, ctx, name)[0m
[0;32m     65[0m     [1;32melse[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0;32m     66[0m       [0mmessage[0m [1;33m=[0m [0me[0m[1;33m.[0m[0mmessage[0m[1;33m[0m[1;33m[0m[0m
[1;32m---> 67[1;33m     [0msix[0m[1;33m.[0m[0mraise_from[0m[1;33m([0m[0mcore[0m[1;33m.[0m[0m_status_to_exception[0m[1;33m([0m[0me[0m[1;33m.[0m[0mcode[0m[1;33m,[0m [0mmessage[0m[1;33m)[0m[1;33m,[0m [1;32mNone[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m     68[0m   [1;32mexcept[0m [0mTypeError[0m [1;32mas[0m [0me[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0;32m     69[0m     keras_symbolic_tensors = [

[1;32m~\miniconda3\envs\tensorflow\lib\site-packages\six.py[0m in [0;36mraise_from[1;34m(value, from_value)[0m

[1;31mInternalError[0m:  Blas GEMM launch failed : a.shape=(3, 50), b.shape=(50, 200), m=3, n=200, k=50
	 [[{{node lstm_1/while/body/_1/MatMul_1}}]] [Op:__inference_keras_scratch_graph_1472]

Function call stack:
keras_scratch_graph

InternalError:  Blas GEMM launch failed : a.shape=(3, 50), b.shape=(50, 200), m=3, n=200, k=50
	 [[{{node lstm_1/while/body/_1/MatMul_1}}]] [Op:__inference_keras_scratch_graph_1472]

Function call stack:
keras_scratch_graph


